<launch>

     <!-- bigman.launch with robot state publisher --> 
     <include file="$(find bigman_urdf)/launch/bigman.launch"/>

     <!-- yarp_ros_joint_state_publisher -->	
     <node pkg="yarp_ros_joint_state_publisher" type="yarp_ros_joint_state_publisher_node" name="yarp_ros_joint_state_publisher" output="screen">
       <param name="robot_name" type="string" value="bigman" />
       <param name="urdf_path" type="string" value="$(find bigman_urdf)/urdf/bigman.urdf" />
       <param name="srdf_path" type="string" value="$(find bigman_srdf)/srdf/bigman.srdf" />
     </node>

     <!-- robot_state_publisher_ext -->	
     <node pkg="robot_state_publisher_ext" type="robot_state_publisher_ext_node" name="robot_state_publisher_ext" output="screen" >
       <param name="robot_name" type="string" value="bigman" />
       <param name="urdf_path" type="string" value="$(find bigman_urdf)/urdf/bigman.urdf" />
       <param name="srdf_path" type="string" value="$(find bigman_srdf)/srdf/bigman.srdf" />
     </node>


</launch>
